ecleankernel/file.py

```diff
@@ -111,7 +111,10 @@ def __init__(self,
         super().__init__(path, KernelFileType.KERNEL)
         self.internal_version = self.read_internal_version()
 
-    def decompress_raw(self, f: typing.IO[bytes]) -> bytes:
+    def decompress_raw(self,
+                       f: typing.IO[bytes],
+                       size: int = -1
```

Comment:
The Pythonic way of doing that would be:

```suggestion
                       size: typing.Optional[int] = None,
```